docs/content/en/docs/getting-started/production-environment/vsphere-getstarted.md

```diff
@@ -177,40 +178,54 @@ Follow these steps if you want to use your initial cluster to create and manage
    export EKSA_LICENSE='my-license-here'
    ```
 
-1. Create a workload cluster
+1. Create a workload cluster in one of the following ways:
+   
+   * **GitOps**: Recommended for more permanent cluster configurations. See [Manage cluster with GitOps]({{< relref "/docs/tasks/cluster/cluster-flux" >}}) for details.
```

Comment:
We should also add the note to be sure to specify the `namespace` for all EKS-A objects when using Gitops to create new workload clusters. Even if using the `default` namespace, indicate that by specifying `namespace: default` on those objects.

Reply:
Thanks I forgot that.